ServerCore/Pages/Resources/pd2023/FAQPartial.cshtml

```diff
@@ -97,6 +97,20 @@ else
             The best way to win is to solve puzzles quickly so that you get to the final metapuzzle. The best way to accomplish this is with teamwork. Many teams prefer to work individually on the easier puzzles and work as a group on the harder puzzles. If you get stuck, ask for help or switch to another puzzle. Make sure you keep track of which puzzles you've already solved and confirmed using the answer submission system.
         </p>
 
+        <h3>
+            Will food be provided?
+        </h3>
+        <p>
+            Pizza will be provided for all teams.  If you or a team member cannot eat pizza, please reach out to us and we will provide an alternative.
+        </p>
+
+        <h3>
+            When and how do we get our food?
+        </h3>
+        <p>
+            Watch your inbox! Your team will receive an email letting you know when food is ready with instructions on where to go to pick it up.
```

Comment:
Will this mail come from puzzhunt or will it come from the bulk mailer on the site? If the latter, we need to also encourage them to monitor junk mail here.

Reply:
Great question.  I'll check with Morgan/Megan today.  I expect it's the bulk mailer.